src/components/MapEmbed.js

```diff
@@ -18,15 +21,17 @@ import { isSinglePostInEditor } from '../helpers/globals';
  * @return {JSX.Element} The rendered React component.
  */
 const MapEmbed = (props) => {
+	const isAdmin = select('core')?.canUser('create', 'posts');
+	const isPostEditor = select('core/edit-post') ? true : false;
```

Comment:
would do something like `const isPostEditor = Boolean(select('core/edit-post'));` so it looks a bit cleaner.